src/components/Material/AnalysisContent.tsx

```
@@ -0,0 +1,107 @@
+  analysis: Analysis | null
+  folderId: number
+}
+
+function AnalysisContent({ analysis, folderId }: AnalysisContentProps) {
+  if (!analysis)
+    return (
+      <div className="flex justify-center items-center">
+        분석 결과가 존재하지 않습니다.
+      </div>
+    )
+  const getColor = useFolderColorStore((state) => state.getColor)
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)